### PR TITLE
Switch to Jest based expectations

### PR DESCRIPTION
**Pull request by orta**
This makes it feasible for us to use JSON snapshots 

👍

### CODE REVIEW

**Comment by orta** [comment]
OK, this is good to be look at now 👍

File: test/lib/apis/galaxy.js [hunk 1]
```diff
@@ -43,29 +43,35 @@ describe('APIs', () => {
...
     });
 
     it('rejects request errors', () => {
       const request = sinon.stub().yields(new Error('bad'));
       fetch.__Rewire__('request', request);
 
-      return expect(galaxy('foo/bar')).to.be.rejectedWith('bad');
+      return galaxy('foo/bar').catch(e => {
+        expect(e.message).toEqual('bad');
+      });
```
Comment (alloy) [inline test/lib/apis/galaxy.js]: This is not the same test, as it would silently pass if the promise did not reject.

**Comment by orta** [comment]
Alright, that's them resolved.

**Comment by alloy** [comment]
Looks good!

**Pull request merged**